### ISSUE TITLE
inference (centermask_demo) showing mask data only for very few objects.

### ISSUE THREAD

**Issue by omerbrandis** [youngwanLEE/CenterMask#61]
hello, 

I've trained centermask on my own dataset.
used several versions of centermask_v configs (19,39,99).
(all of them stalled at around loss = 0.65 .)

I've executed centermask_demo on the training set to see what the cnn has learned, 
I see that some objects are identified (bbox), but most of them do not display mask information.
(the annotation files are probably not the problem, i've used then with other cnns and got better results)

here's an example, as you can see , several apples are identified, but only one of them has mask data.
![img1_result](https://user-images.githubusercontent.com/3157363/109410047-1a6cd380-79a0-11eb-8d53-082d9ffc8124.jpg)

please advise,
Omer.

**Comment by lijianaiml** [comment]
Hi,i have occur the same question,have you solve it now?
@ omerbrandis

**Comment by lijianaiml** [comment]
@omerbrandis
